### PR TITLE
docs: add kommander 1.1.0 rc3 release notes

### PR DESCRIPTION
**Pull request by juliangieseke**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 1]
```diff
@@ -50,6 +50,33 @@ Template:
...
+#### Bug Fixes
+
+- Smaller UX Bugs and Improvements
+
+#### Component Versions
+
+- Addon: `1.1.0-34`
+- Chart: `0.8.15`
+- auto-provisioning (yakcl): `0.1.0`
+- kommaner-deferation (yakcl): `0.1.0`
```
Comment (juliangieseke) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:71]:
```suggestion
- kommaner-federation (yakcl): `0.1.0`
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 2]
```diff
@@ -50,6 +50,33 @@ Template:
...
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.15.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+
+#### Bug Fixes
+
+- Smaller UX Bugs and Improvements
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:64]: A lot of component changes but they all fell under this?

Comment (juliangieseke) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:64]: I havent seen any noteworthy changes in https://jira.d2iq.com/projects/D2IQ/versions/41236 - thats completely IMHO tho, happy to discuss/change if you want something listed. 

We should make all EM/PO/PM sign off the release notes for the stable release anyway, but I figured thats not that important for betas. 🤷

Comment (samvantran) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:64]: sgtm 👍

**Pull request merged**